server/controller/login_controller.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable linebreak-style */
+import { getSingleUser } from '../helper/userHelper';
+
+const wrongDataStatus = (res, message) => {
+  res.status(400).send({
+    status: 400,
+    message,
+  });
+};
+
+export default function login(req, res) {
+  if (req.body.email === undefined) {
+    res.status(400).send({
+      status: 400,
+      message: 'the email is required, please provide it before proceeding',
+    });
+  } else if (req.body.password === undefined) {
+    wrongDataStatus(res, 'the password is required, please provide it before proceeding');
```

Comment:
A sentence should start with a capital letter, eg `The password is required`, make the message very short and descriptive. The one you provided above is very long. You can have it like have shown you here.